DQM/Integration/python/clients/sistriplas_dqm_sourceclient-live_cfg.py

```diff
@@ -1,8 +1,13 @@
 from __future__ import print_function
 import FWCore.ParameterSet.Config as cms
 
-from Configuration.Eras.Era_Run3_cff import Run3
-process = cms.Process( "sistriplaserDQMLive", Run3 )
+if 'runkey=hi_run' in sys.argv:
```

Comment:
off-topic but the LAS system doesn't exist since run1 :)
Wondering if this is still being run ....

Reply:
It's not: https://cmsweb.cern.ch/dqm/dqm-square/ my bad :D
I'll revert the changes for this client. Thanks for spotting this!

Reply:
you can keep them, and we target this for deletion at the next occasion.

Reply:
ok cool! thanks!